src/tables/tableRows.tsx

```diff
@@ -366,7 +366,9 @@ export function generateUnitRows(
               value={unitId}
               data-testid={fieldID}
             />
-            <span className="p-checkbox__label" id={ariaLabeledBy}></span>
+            <span className="p-checkbox__label" id={ariaLabeledBy}>
```

Comment:
I made this change while trying to debug why it couldn't click on the checkbox and have left it here as it's more accessible (previously the checkbox had not accessible label).